### PR TITLE
Added .ruby-version file to lock ruby version in the project

### PR DESCRIPTION
**Pull request by anuragajwani**
# Problem
Ruby version is not locked in the project

# Solution
Add .ruby-version for rvm to set the correct ruby version used in the project

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.4.5
```
Comment (joninvski) [inline .ruby-version]: technically this is not needed as it is already in `test/Gemfile` (also versions are different)

If you want  you can remove it from there and leave this `.ruby_version`. They are two different ways to do the same thing.

Comment (anuragajwani) [reply .ruby-version]: The Gemfile doesn't seem to cantain any references to the ruby version:
```
# frozen_string_literal: true
source "https://rubygems.org"

gem 'monster', :git => 'git@bitbucket.org:onfido/monster.git', :tag => '0.8.26'
gem 'cucumber', :git => 'https://github.com/cucumber/cucumber-ruby.git', :ref => '085be64'
gem 'cucumber-api', :git => 'https://github.com/cristian-m-vasile/cucumber-api.git'
gem 'rake'
gem 'selenium-webdriver'
gem 'browserstack-local'
```

The `Gemfile.lock` has the version of bundler used but not the ruby version either. Am I missing something?

Comment (bartzet) [reply .ruby-version]: This Gemfile will be soon 🔥 as I'm replacing the last Ruby UI test.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-735-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://735-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-735-pr-onfido-sdk-ui-onfido.surge.sh

File: .ruby-version (hunk 1 above)
Comment (anuragajwani) [reply .ruby-version]: cool I guess no point for this then

**Pull request closed without merging**